### DISCUSSION TITLE
Cannot Understand How To Get This Working On Windows

### DISCUSSION THREAD

**Discussion by TECHNETWORKINGSOLUTIONS** [hrfee/jfa-go#107] [Q&A]
Hi, I’m having trouble understanding how to get this setup on my windows server and I couldn’t find any information on it. 
Can Anyone assist me with the setup process that would be great 
—Cheers

**Comment by hrfee** [comment]
Download a stable version [here](https://github.com/hrfee/jfa-go/releases/download/v0.3.6/jfa-go_0.3.6_TrayIcon_Windows_x86_64.zip), or a nightly version [here](https://builds.hrfee.dev/repo/hrfee/jfa-go/build/a83dbcf3ab83851ba2db43c73d885b6eaaadd45e/jfa-go_0.0.0-a83dbcf_TrayIcon_Windows_x86_64.zip). Unzip the file, and run `jfa-go.exe`. A little tray icon should appear in the bottom right , like this:

![20210525_16h29m24s_grim](https://user-images.githubusercontent.com/11923517/119525463-70220380-bd76-11eb-83a8-49819b762acc.png)

It it's there, navigate to `localhost:8056` in your browser to start the set-up.

**Reply by TECHNETWORKINGSOLUTIONS** [reply]
Ok now every time I try to run jfa it starts for a few secs then closes by its self?

**Reply by hrfee** [reply]
Try deleting `C:\Users\<your username>\AppData\Roaming\jfa-go`, then running again. On the first run of jfa-go, an example config file is placed there, so if you then close and re-open the program, it'll try to use the example config which isn't valid, and so it crashes.

**Reply by TECHNETWORKINGSOLUTIONS** [reply]
I am still receiving the same results when i delete it i try to open it again and continues to do the same thing and it just adds another config file

**Reply by hrfee** [reply]
Are you sure that the app is closing every time? The  tray icon might just be getting moved to the overflow window. Try accessing `localhost:8056` as soon as the icon appears. Also, if you're running from a command line, the app will always look like it instantly closes, as thats how GUI apps work on windows.

**Reply by TECHNETWORKINGSOLUTIONS** [reply]
alright hold up because it just loaded actually but i have another question for the setup because i tried the setup and it connected to my server fine but after config it wouldn't load with the address of localhost:8056 anymore

**Reply by hrfee** [reply]
Right-click on the tray icon and press "Open logs", then send the content of the log file that opens. Make sure there's no sensitive information in there, though.

If the tray icon disappears before you can, the file is located at `C:\Users\<your username>\AppData\Local\Temp\jfa-go.log`.
